usr/sbin/rear

```diff
@@ -55,7 +55,7 @@ readonly INITIAL_BASH_FLAGS_AND_OPTIONS_COMMANDS="$( get_bash_flags_and_options_
 # Versioning
 readonly PRODUCT="Relax-and-Recover"
 readonly PROGRAM=${0##*/}
-readonly VERSION=2.5
+readonly VERSION=2.5.rubrik
```

Comment:
@DamaniN 
please remove that change.
We never do that during development of the next ReaR version.
I guess when VESION changes this way there could be
various kind of unexpected weird side effects.

Reply:
Reverted back to original per this note.